packages/sampler/stories/qa/Remeasurable.js

```diff
@@ -30,6 +30,7 @@ storiesOf('Remeasurable', module)
 		() => (
 			<Scroller style={{height: ri.unit(399, 'rem'), width: ri.unit(501, 'rem')}}>
 				<NoUpdate>
+					<Item marqueeOn="render">MARQUEEONRENDER ABCDE</Item>
```

Comment:
@LucieRoy Can you confirm these changes to the QA sample don't cause any of the test cases to change?